crates/resvg/src/image.rs

```diff
@@ -87,7 +87,22 @@ mod raster_images {
 
         let options = DecoderOptions::default().jpeg_set_out_colorspace(ColorSpace::RGBA);
         let mut decoder = zune_jpeg::JpegDecoder::new_with_options(data, options);
-        let img_data = decoder.decode().ok()?;
+        decoder.decode_headers().ok()?;
+        let output_cs = decoder.get_output_colorspace()?;
+
+        let img_data = {
+            let data = decoder.decode().ok()?;
+            match output_cs {
+                ColorSpace::RGBA => data,
+                // For some reason zune jpeg won't coerce luma JPEGs into RGBA, so we do it manually.
```

Comment:
This ideally needs a link to an upstream issue.

Reply:
From what I understand it's not really an issue, it's just that zune_jpeg doesn't guarantee that all input types can be converted to RGBA. See: https://docs.rs/zune-core/latest/zune_core/options/struct.DecoderOptions.html#method.jpeg_set_out_colorspace